src/bundle/Resources/public/js/CKEditor/special-characters/special-characters.js

```diff
@@ -0,0 +1,14 @@
+import SpecialCharacters from '@ckeditor/ckeditor5-special-characters/src/specialcharacters';
+
+export default class IbexaSpecialCharacters extends SpecialCharacters {
+    _updateGrid(currentGroupName, gridView) {
+        super._updateGrid(currentGroupName, gridView);
+
+        gridView.tiles._items.forEach((tile) => {
```

Comment:
```suggestion
        gridView.tiles.forEach((tile) => {
```
as `tiles` is a Collection this should be more accurate.